flatdata-cpp/test/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-find_package(GTest CONFIG QUIET)
```

Comment:
Why this change?

Reply:
To be able to find the package, if it was installed system-wide as a module.